docs/FAQ.md

```diff
@@ -0,0 +1,98 @@
+
+**Section 1: [FAQ](#faq)**
+
+*Performance*
+* [What type of GPU is supported?](#What-type-of-GPU-is-supported)
+* [Which Execution Provider will give me the best performance?](#Which-Execution-Provider-will-give-me-the-best-performance)
+* [Do the GPU builds support quantized models?](#Do-the-GPU-builds-support-quantized-models)
+
+*API*
+* [Does the Python API add any overhead to performance?](#Does-the-Python-API-add-any-overhead-to-performance)
+* [How do I turn on verbose mode logging in Python?](#How-do-I-turn-on-verbose-mode-logging-in-Python)
+* [How do I infer models that have multiple inputs and outputs using the C/C++ API?](#How-do-I-infer-models-that-have-multiple-inputs-and-outputs-using-the-CC-API)
+* [How do I force single threaded execution mode in ORT?](#How-do-I-force-single-threaded-execution-mode-in-ORT)
+
+**Section 2: [Troubleshooting](#troubleshooting)**
+* [Performance Troubleshooting Checklist](#performance-troubleshooting-checklist)
+* [Why is my model running slower on GPU than CPU?](#Why-is-my-model-running-slower-on-GPU-than-CPU)
+* [My converted Tensorflow model is slow - why?](#My-converted-Tensorflow-model-is-slow---why)
+* [My PyTorch model is running slower on GPU - why?](#My-PyTorch-model-is-running-slower-on-GPU---why)
+* [I can't build the xyz execution provider.](#I-cant-build-the-xyz-execution-provider)
+
+***
+# FAQ 
+
+## What type of GPU is supported? 
+The default ONNX Runtime GPU execution provider is for NVIDIA GPUs with CUDA. This build is published on Nuget: [Microsoft.ML.Onnxruntime.Gpu](https://www.nuget.org/packages/Microsoft.ML.OnnxRuntime.gpu) and PyPi: [onnxruntime-gpu](https://pypi.org/project/onnxruntime-gpu). To build from source, see [BUILD.md](https://github.com/microsoft/onnxruntime/blob/master/BUILD.md#cuda). 
+
+Additionally, for NVIDIA GPUs you can build the [TensorRT execution provider](./execution_providers/TensorRT-ExecutionProvider.md) following [these instructions from BUILD.md](./../BUILD.md#tensorrt)
+
+If your deployment target is Windows, we recommend using the [DirectML execution provider](./execution_providers/DirectML-ExecutionProvider.md) for optimal performance and compatibility with a broad set of GPUs.  See the build instructions [here](./../BUILD.md#directml).
+
+AMD GPU support is in progress.
+
+## Which Execution Provider will give me the best performance? 
+Performance is dependent on the specific model you're trying to run, the session and run options you've selected, and of course, your specific hardware target. Please refer to the [Performance Tuning](./ONNX_Runtime_Perf_Tuning.md) page for guidance on how to performance tune ONNX Runtime for your model. You can use the [ONNX Go Live - OLive](https://github.com/microsoft/OLive) tool to help test and identify the best configuration and execution provider to use.  Below you'll find some more information that may be helpful to select the right Execution Provider.
+
+### CUDA (Default GPU) vs CPU?
```

Comment:
It's unclear why this is a FAQ. GPU vs CPU is a question that is not specific to ORT and customers generally know which one they are going to use.

Reply:
It was meant to capture our guidance on which EP to choose and things to keep in mind with limitations of each....this particular comparison notes the limitations with lack of full operator support on CUDA compared to the full support with the CPU implementation. I moved this whole section from FAQ to the perf tuning page in a new iteration...if you still think it sounds off there, I'll revise in the new draft.